README.rst

```diff
@@ -155,3 +155,75 @@ Using your normal views over a websocket connection
       ),
    })
 
+
+Creating fully-functional custom Consumers
+------------------------------------------
+
+This package offers Django Rest Framework capabilities via mixins. To utilize these mixins, one must inherit from the GenericAsyncAPIConsumer. One may use the same exact querysets and serializer_classes utilized in their DRF Views. However, one must import djangochannelsrestframework's permissions, which provides the standard AllowAny and IsAuthenticated permissions.
+
+from . import models
+from . import serializers
+from djangochannelsrestframework import permissions
+from djangochannelsrestframework.generics import GenericAsyncAPIConsumer
+from djangochannelsrestframework.mixins import (
+    ListModelMixin,
+    PatchModelMixin,
+    UpdateModelMixin,
+    CreateModelMixin,
+    DeleteModelMixin,
+)
+
+class LiveConsumer(ListModelMixin, GenericAsyncAPIConsumer):
+    queryset = models.Test.objects.all()
+    serializer_class = serializers.TestSerializer
+    permission_classes = (permissions.IsAuthenticated,)
+
+    extracted_data, extracted_status = await self.list()
```

Comment:
maybe have a separate subsection were you talk about using these methods from within other methods. In this section i think it would be better to so how you would send a message at a consumer that has the  ListModelMixin.

```
{action: "list", "request_id": 42}
```